esmvalcore/_data_finder.py

```diff
@@ -95,16 +103,29 @@ def get_start_end_year(filename):
                 time = cube.coord('time')
             except iris.exceptions.CoordinateNotFoundError:
                 continue
-            start_year = time.cell(0).point.year
-            end_year = time.cell(-1).point.year
+            start_point = time.cell(0).point
+            start_datetime = datetime.datetime(
```

Comment:
Why this and not directly use the point?   It is not working with the `isodate` module? The class that iris is using for this is usually close enough to the one in datetime for this replacement to work

Reply:
I think I started using another parser that was not working with the cftime Datetime object. But this one seems to work.